astutil.go

```diff
@@ -165,6 +170,13 @@ func doCaseClause(clause []ast.Expr, body []ast.Stmt) *ast.CaseClause {
 	}
 }
 
+func doIntegerLit(value int64) *ast.BasicLit {
+	return &ast.BasicLit{
+		Kind:  token.INT,
+		Value: fmt.Sprintf("%d", value),
```

Comment:
Use strconv.Itoa here. We should get rid of these Sprintf("%d".. conversions. This is usually my mistake ;-)